dblookupext/esdtSupply/logsProcessor.go

```diff
@@ -127,20 +129,39 @@ func (lp *logsProcessor) processEvent(txLog *transaction.Event, supplies map[str
 
 	valueFromEvent := big.NewInt(0).SetBytes(txLog.Topics[2])
 
-	tokenIDStr := string(tokenIdentifier)
-	tokenSupply, found := supplies[tokenIDStr]
+	err := lp.updateOrCreateTokenSupply(tokenIdentifier, valueFromEvent, string(txLog.Identifier), supplies, isRevert)
+	if err != nil {
+		return err
+	}
+
+	if isESDTFungible {
+		return nil
+	}
+
+	collectionIdentifier := txLog.Topics[0]
```

Comment:
same as `tokenIdentifier` declared on L120

Reply:
tokenIdentifier is updated `if len(txLog.Topics[1]) != 0`

Reply:
right